index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Microverse Day 2 Project</title>
+</head>
+<body>
+    <div class="header">
+        <h1>Hello Microverse!!!</h1>
+      </div>
+    
+    
```

Comment:
- [ ] These empty lines seem to serve no purpose here and I think it will be a lot nicer to remove them. This is best practice and conventional to ensure a consistently readble code base.